tests/modules/mesoscope_splitting_cli/conftest.py

```diff
@@ -175,7 +239,7 @@ def zstack_fixture(zstack_metadata_fixture,
                 zz = zz
             exp_id = z_to_exp_id_fixture[z_pair][zz]
             expected_path = tmp_dir / f'{exp_id}_expected_z_stack.h5'
-            data = rng.integers(0, 10*zz, (n_pages, 24, 24))
+            data = rng.integers(0, 10*zz, (n_pages, 25, 24))
```

Comment:
Why the change to 25? Same with the ones below. If this relates to another piece of input data, you should likely set it with that related value or have a single variable for the values.

Reply:
Those numbers actually have nothing to do with each other. I changed the so that the images weren't squares, just in case I did something wrong in converting from `(x, y)` to `(row, col)` in the code being tested.

I've parameterized them  and added a comment explaining that they are arbitrary.